src/HyperFormula.ts

```diff
@@ -2730,20 +2789,22 @@ export class HyperFormula implements TypedEmitter {
    *
    * @category Helpers
    */
-  public getCellPrecedents(address: SimpleCellAddress | AbsoluteCellRange): (AbsoluteCellRange | SimpleCellAddress)[] {
+  public getCellPrecedents(address: SimpleCellAddress | SimpleCellRange): (SimpleCellRange | SimpleCellAddress)[] {
```

Comment:
@kirszenbaum Same here, we may show another way to use this method in the `@example` tag

Reply:
@wojciechczerniak - sure, I added a note to the backlog: https://github.com/handsontable/hyperformula/projects/1#card-63587403